### DISCUSSION TITLE
Cleanup: parent + child slots both on Biosample - do we need them?

### DISCUSSION THREAD

**Discussion by bmeluch** [microbiomedata/nmdc-schema#2109] [General]
There are 5 slots on Biosample that I think are only meant to be used for grouping slots together, not to actually hold data:
* [core_field](https://microbiomedata.github.io/berkeley-schema-fy24/core_field/)
* [environment_field](https://microbiomedata.github.io/berkeley-schema-fy24/environment_field/)
* [investigation_field](https://microbiomedata.github.io/berkeley-schema-fy24/investigation_field/)
* [nucleic_acid_sequence_source_field](https://microbiomedata.github.io/berkeley-schema-fy24/nucleic_acid_sequence_source_field/)
* [sequencing_field](https://microbiomedata.github.io/berkeley-schema-fy24/sequencing_field/)

The first four have child slots on Biosample, `sequencing_field` children are on `LibraryPreparation`, `NucleotideSequencing`, and nowhere.
Montana mentioned that `core_field` is probably a relic of how MIxS is imported and may be changing anyway.

So, questions - 
1. Can we take these 5 slots off of Biosample, since it doesn't make sense to populate them? 
2. Are these slots helpful in other ways, do we need them at all?

**Comment by turbomam** [comment]
I am looking at this again now as part of my subsets review. I don't think there's any reason to keep them associated with `Biosample`

If they are being used to group other slots, maybe they should be marked [is_grouping_slot](https://linkml.io/linkml-model/latest/docs/is_grouping_slot/) . (Question to self: Or is that to indicate that they are the target of [slot_group](https://linkml.io/linkml-model/latest/docs/slot_group/) assertions?)

we do have the following **empty** [subsets](https://microbiomedata.github.io/nmdc-schema/#subsets), which seem redundant
- environment
- investigation
- nucleic acid sequence source
- sequencing

I don't see any `core` subset
